app/elements/calendar-app.html

```diff
@@ -139,9 +148,14 @@
           fn();
         }
 
-        if (event.status === "FINISHED") {
-          element.css('border', '#15D115 5px solid');
-        }
+        var link = $('<div>')
+            .addClass('event-finisher')
```

Comment:
@GijsWeterings this should be the `trafficlight`. Could you cherry-pick the trafficlight onto this branch and insert it here?